modules/policy/engine/src/main/java/org/eclipse/sw360/antenna/policy/engine/PolicyViolation.java

```diff
@@ -25,11 +24,11 @@
  */
 public class PolicyViolation {
     private Rule rule;
-    private Collection<ThirdPartyArtifact> failedArtifacts = new HashSet<>();
+    private Collection<ThirdPartyArtifact> failingArtifacts = new HashSet<>();
```

Comment:
Single renamings like this should be separated out to another commit.

Reply:
This is really an interesting strictness. I once learned that it is a good practice to clean up a piece of code you are working on in a change in order to improve the codebase step by step. Should be part of our discussion, my opinion here is, that if it is only improving understandability or making things more consistent is not a special commit, if the code is refactored structurally, this is of course a new commit.

Reply:
I guess this is related to the discussion whether change == commit or change == PR. Of course, if you come across a sensible renaming while working on something else, you should be free to do that in the same PR, but I'd still vote for doing it in a separate commit, so that functional and non-functional changes are separated. But I agree to discuss that separately.